webapp/app/not-found.tsx

```diff
@@ -1,3 +1,33 @@
+'use client'
+
+import 'styles/globals.css'
+import { ExclamationMark } from 'components/icons/exclamationMark'
+import Image from 'next/image'
+
+import Error404 from './_images/404.svg'
+
 export default function NotFound() {
-  return <div>404</div>
+  return (
+    <div className="flex h-screen">
+      <Image
+        alt="404"
+        className="absolute inset-0 -top-64 m-auto w-72 md:w-fit"
+        src={Error404}
+      />
+      <div className="z-10 m-auto flex flex-col items-center gap-4">
+        <ExclamationMark />
+        <div className="text-center">
+          <h1 className="text-lg font-medium text-neutral-950">
+            Oops! Page not found
```

Comment:
Can we apply translations to these error messages?

Reply:
For the 404, hitting this page may mean that the user requested the page `https://app.hemi.xyz/sarasa`

As locales are part of the URL, what locale would you apply? I think for this particular case, English is just fine.

Now, this is a global `not-found`. I think you can add this file on different route segments. So if a `not-found.tsx` was created below `app/[locale]/` folder, then we would get access to translations. Note that the global not-found will still be used when requesting `https://app.hemi.xyz/sarasa`, but if requesting `https://app.hemi.xyz/en/sarasa`, then we would get the 404 page with the locale defined.

Reply:
@gabmontes I explained about the translations in the PR description.

@gndelia Do you want me to add a `not-found.tsx` file on every route segments?

Reply:
> Do you want me to add a not-found.tsx file on every route segments?

I think the global one (without translation) + one under `/[locale]`  (with translations) should work well for all URLs.

Reply:
☝🏼 that's what should be done here.

Reply:
done